src/string-calculator.ts

```diff
@@ -132,56 +117,39 @@ export class StringCalculator {
     private getCustomSeparator(): string {
         let beginOfSep: number = this.numberList.indexOf("//") + 2;
         let endOfSep: number = this.numberList.indexOf("\n");
-        if (beginOfSep != -1) {
+        if (beginOfSep) {
             return this.numberList.substring(beginOfSep, endOfSep);
         }
 
         return "";
     }
 
-    private checkForErrors(number: string):ErrorCode {
-        let err: ErrorCode;
-        
-        err = this.isNumberNegative(number);
-        if(err) {
-            return err;
-        }
-        err = this.isNumberMissed(number);
-        if(err) {
-            return err;
-        }
-        err = this.isNumberNumeric(number);
-        if(err) {
-            return err;
-        }
-        return null;
+    private checkForErrors(number: string) {
+        this.isNumberNegative(number);
+        this.isNumberMissed(number);
+        this.isNumberNumeric(number);
     }
 
-    private isNumberNegative(number: string): ErrorCode {
+    private isNumberNegative(number: string) {
         if (number.startsWith("-")) {
-            //throw new NegativeNumberException();
-            return ErrorCode.NegativeNumberErrorCode;
+            throw new ErrorCodeExeption(ErrorCode.NegativeNumber);
         }
         return null;
```

Comment:
You don't need a return statement here. This is just style but in this case it'll make the code cleaner.